### ISSUE TITLE
Flutter Webview_flutter  - Search keypad does not show for input fields

### ISSUE THREAD

**Issue by f1mert** [flutter/flutter#56515]
open a website with webview_flutter plugin such as：
`<input type="search" />`
if open this with android webview, it will show an search button on rightbottom at the first time,
but it's not worked in flutter .

**Comment by TahaTesser** [comment]
Issue does reproduce 
When clicking `<input type="search" />` in `webview_flutter` in Navigation Delegate example
Keyboard action button is search button for a second than it changes to enter key
No such issue in native webview 

<details> 
<summary>Code Sample</summary> 

``` 
const String kNavigationExamplePage = '''
<!DOCTYPE html><html>
<head><title>Navigation Delegate Example</title></head>
<body>
<input type="search" />
<p>
The navigation delegate is set to block navigation to the youtube website.
</p>
<ul>
<ul><a href="https://www.youtube.com/">https://www.youtube.com/</a></ul>
<ul><a href="https://www.google.com/">https://www.google.com/</a></ul>
</ul>
</body>
</html>
''';
``` 

</details> 

<details> 
<summary>flutter doctor -v</summary> 

``` 
[✓] Flutter (Channel dev, 1.18.0-12.0.pre, on Mac OS X 10.15.4 19E287, locale en-GB)
    • Flutter version 1.18.0-12.0.pre at /Users/tahatesser/Code/flutter_dev
    • Framework revision c2b7342ca4 (21 hours ago), 2020-05-06 23:16:03 +0800
    • Engine revision 33d2367950
    • Dart version 2.9.0 (build 2.9.0-5.0.dev 9c94f08410)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/tahatesser/Code/SDK
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/tahatesser/Code/SDK
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4.1, Build version 11E503a
    • CocoaPods version 1.9.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] VS Code (version 1.44.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.10.1

[✓] Connected device (5 available)
    • SM M305F      • 32003c30dc19668f          • android-arm64  • Android 10 (API 29)
    • Taha’s iPhone • 00008020-001059882212002E • ios            • iOS 13.4.1
    • macOS         • macOS                     • darwin-x64     • Mac OS X 10.15.4 19E287
    • Web Server    • web-server                • web-javascript • Flutter Tools
    • Chrome        • chrome                    • web-javascript • Google Chrome 81.0.4044.138

• No issues found!



``` 

</details>

**Comment by f1mert** [comment]
android &amp; IOS&nbsp; with HUAWEI P20






------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Taha Tesser"<notifications@github.com&gt;; 
发送时间: 2020年5月7日(星期四) 晚上10:05
收件人: "flutter/flutter"<flutter@noreply.github.com&gt;; 
抄送: "独一无二"<734219084@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [flutter/flutter] Flutter Webview_flutter  - Search keypad does not show for input fields (#56515)





 
Issue does reproduce
 When clicking <input type="search" /&gt; in webview_flutter in Navigation Delegate example
 Keyboard action button is search button for a second than it changes to enter key
 No such issue in native webview
   Code Sample  const String kNavigationExamplePage = ''' <!DOCTYPE html&gt;<html&gt; <head&gt;<title&gt;Navigation Delegate Example</title&gt;</head&gt; <body&gt; <input type="search" /&gt; <p&gt; The navigation delegate is set to block navigation to the youtube website. </p&gt; <ul&gt; <ul&gt;<a href="https://www.youtube.com/"&gt;https://www.youtube.com/</a&gt;</ul&gt; <ul&gt;<a href="https://www.google.com/"&gt;https://www.google.com/</a&gt;</ul&gt; </ul&gt; </body&gt; </html&gt; ''';      flutter doctor -v  [✓] Flutter (Channel dev, 1.18.0-12.0.pre, on Mac OS X 10.15.4 19E287, locale en-GB)     • Flutter version 1.18.0-12.0.pre at /Users/tahatesser/Code/flutter_dev     • Framework revision c2b7342ca4 (21 hours ago), 2020-05-06 23:16:03 +0800     • Engine revision 33d2367950     • Dart version 2.9.0 (build 2.9.0-5.0.dev 9c94f08410)   [✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)     • Android SDK at /Users/tahatesser/Code/SDK     • Platform android-29, build-tools 29.0.3     • ANDROID_HOME = /Users/tahatesser/Code/SDK     • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java     • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)     • All Android licenses accepted. [✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)     • Xcode at /Applications/Xcode.app/Contents/Developer     • Xcode 11.4.1, Build version 11E503a     • CocoaPods version 1.9.1 [✓] Chrome - develop for the web     • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome [✓] Android Studio (version 3.6)     • Android Studio at /Applications/Android Studio.app/Contents     • Flutter plugin version 45.1.1     • Dart plugin version 192.7761     • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211) [✓] VS Code (version 1.44.2)     • VS Code at /Applications/Visual Studio Code.app/Contents     • Flutter extension version 3.10.1 [✓] Connected device (5 available)     • SM M305F      • 32003c30dc19668f          • android-arm64  • Android 10 (API 29)     • Taha’s iPhone • 00008020-001059882212002E • ios            • iOS 13.4.1     • macOS         • macOS                     • darwin-x64     • Mac OS X 10.15.4 19E287     • Web Server    • web-server                • web-javascript • Flutter Tools     • Chrome        • chrome                    • web-javascript • Google Chrome 81.0.4044.138 • No issues found!

**Comment by libo1223** [comment]
Does anyone have a good way to solve this problem? 
By the way, the search key will change After I operate the keyboard in my phone。
![54988954F7DB5D6F8E70FAE654EDFD97](https://user-images.githubusercontent.com/43235811/83494385-c56f6000-a4e8-11ea-849d-bf290a9ea8c6.jpg)
the s y s

**Comment by blasten** [comment]
This has been fixed in Flutter 1.22/ WebView 1.0 - use hybrid composition in the Flutter WebView plugin.

https://github.com/flutter/plugins/tree/master/packages/webview_flutter#using-hybrid-composition
